common/src/java/org/apache/hadoop/hive/ql/log/PerfLogger.java

```diff
@@ -148,7 +148,7 @@ public static void setPerfLogger(PerfLogger resetPerfLogger) {
    * @param method method or ID that identifies this perf log element.
    */
   @SuppressFBWarnings(value = "NM_METHOD_NAMING_CONVENTION", justification = "Intended")
```

Comment:
Can this be removed now? Same for few methods below.

Reply:
Totally missed these! Thanks 🙏